auth-web/src/components/auth/staff/DissolutionSchedule.vue

```diff
@@ -103,62 +103,45 @@ import { useStaffStore } from '@/stores/staff'
 
 export default defineComponent({
   name: 'DissolutionSchedule',
-  emits: ['update:onHold'],
-  setup (_, { emit }) {
+  setup () {
     const state = reactive({
       menu: false,
-      numberOfBusinesses: 0,
+      numberOfBusinessesEdit: 0,
+      numberOfBusinessesNonEdit: 0,
```

Comment:
I know, this looks weird (having two properties that look similar). I have an explanation though 😆 

The idea is, when the edit panel is open, the schedule summary is going to be reactive to what's being typed in the text field. However, when it is not open, it's going to be what the DB is returning of the current job batch size.

So, I decided to separate those fields. One of the biggest reasons is that, in doing so, I can store the current batch size from the GET call into the nonEdit field. This way, I only need to use the GET call once on mount. Otherwise, I have to do the GET call each time cancel or save is clicked which is really not good haha

Also, this provides so much flexibility and it separates of when the edit panel is open or not.